<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcustomerdialog</class>
 <widget class="QDialog" name="addcustomerdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Customer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="2">
       <widget class="QLineEdit" name="addCustomerDialogLastNameInput"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="addCustomerDialogPhoneInput"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="addCustomerDialogFirstNameInput"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addCustomerDialogAddButton">
        <property name="styleSheet">
         <string notr="true">background: rgb(78, 154, 6)</string>
        </property>
        <property name="text">
         <string>Add Customer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addCustomerDialogCancelButton">
        <property name="styleSheet">
         <string notr="true">background: rgb(164, 0, 0)</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="addCustomerDialogConfirmationMessage">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
